### PR TITLE
Init basic structure

### PR DESCRIPTION
**Pull request by muratcanakcay**

### CODE REVIEW

File: src/Common/common.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include "common.h"
```
Comment (ccpp-apriorit) [inline src/Common/common.cpp]: Please remove Common class with files. Probably you will not have a class called like this in the project. You will create another files here later when you need it.

File: src/Hooks/Hooks_global.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#ifndef HOOKS_GLOBAL_H
```
Comment (ccpp-apriorit) [inline src/Hooks/Hooks_global.h]: Remove this file, it is not needed because your dll doesn't export anything

File: src/Hooks/hooks.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#ifndef HOOKS_H
+#define HOOKS_H
+
+#include "Hooks_global.h"
+
+class HOOKS_EXPORT Hooks
```
Comment (ccpp-apriorit) [inline src/Hooks/hooks.h]: I can almost guarantee that there will be no class called Hooks. Please remove it

File: src/UI/mainwindow.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+class MainWindow : public QMainWindow
+{
+    Q_OBJECT
+
+public:
+    MainWindow(QWidget *parent = nullptr);
+    ~MainWindow();
+
+private:
+    Ui::MainWindow *ui;
```
Comment (ccpp-apriorit) [inline src/UI/mainwindow.h]: Please place it on stack (remove *) and delete destructor. I reccomend you to name class members with prefix m_: `m_ui`

File: src/Common/common.cpp (hunk 1 above)
Comment (muratcanakcay) [reply src/Common/common.cpp]: I removed Common.h and Common.cpp from project

File: src/Hooks/Hooks_global.h (hunk 2 above)
Comment (muratcanakcay) [reply src/Hooks/Hooks_global.h]: Removed

File: src/Hooks/hooks.h (hunk 3 above)
Comment (muratcanakcay) [reply src/Hooks/hooks.h]: Removed Hooks.h and Hooks.cpp

File: src/UI/mainwindow.h (hunk 4 above)
Comment (muratcanakcay) [reply src/UI/mainwindow.h]: renamed ui to m_ui and placed it on stack
deleted destructor

**Review by ccpp-apriorit — APPROVED** [review]

**Pull request merged**